sklearn/utils/estimator_checks.py

```diff
@@ -175,6 +189,7 @@ def _yield_regressor_checks(name, regressor):
     if tags["requires_fit"]:
         yield check_estimators_unfitted
     yield check_non_transformer_estimators_n_iter
+    yield check_constant_features
```

Comment:
Can you test this for all estimators, not only regressors?

Reply:
I placed `check_constant_features` also in the function `_yield_classifier_checks`. Do I need to place it somewhere else?

Reply:
It might be easier to just put it into ``_yield_checks`` which runs it for everything.

Reply:
Perfect, I've just changed it

Reply:
When moving it from `_yield_checks` there are two failed tests for the estimators `GraphicalLassoCV` and `GraphicalLassoCV` with error ` FloatingPointError: The system is too ill-conditioned for this solver`. These errors seem to be due to the presence of a constant feature. I can add an assertion in `check_constant_features` for this error message. If needed I can also change the error message to something more explicit.

Reply:
I added an assertion to check "'The system is too ill-conditioned for this solver".  Right now the coverage test is not passing. I will add a test for this exception.